cccppp/bin/cccppp

```diff
@@ -1 +1 @@
-../src/cccppp
\ No newline at end of file
+/home/shreesh/Desktop/myRepo/toolsub/cccppp/src/interstitial
```

Comment:
This change should be reverted, as it points to some path on your local machine.